manifests/c/CryptoPro/Chromium-Gost/108.0.5359.94/CryptoPro.Chromium-Gost.locale.ru-RU.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: CryptoPro.Chromium-Gost
+PackageVersion: 108.0.5359.94
+PackageLocale: ru-RU
+Publisher: CryptoPro
+PackageName: Chromium-Gost
+License: MIT
+Copyright: Copyright 2022 The Chromium-Gost Authors. All rights reserved.
+ShortDescription: Chromium with GOST crypto algorithm support

```

```suggestion
ShortDescription: Chromium с поддержкой алгоритмов ГОСТ
```
As Russian locale is used.